osc-staging.py

```diff
@@ -157,6 +158,11 @@ def do_staging(self, subcmd, opts, *args):
         changed from state new or review more than 3 days ago will be removed.
 
     "list" will list/supersede requests for ring packages or all if no rings.
+        As an example, the following would list all packages which have received a
+        positive review from a member of the licensedigger group or the factory-auto
+        one
+
+        list --match-filter "review[(@by_group='factory-auto' or @by_group='licensedigger') and @state='accepted']"
```

Comment:
Should we also add `state/@name='review'` to the filter here?

Reply:
sure, fixed. thank you!